src/openvic-simulation/economy/production/ResourceGatheringOperation.cpp

```diff
@@ -350,8 +349,8 @@ void ResourceGatheringOperation::pay_employees(
 
 	total_owner_income_cache = 0;
 	total_employee_income_cache = 0;
-	if (revenue <= 0 || total_worker_count_in_province <= 0) {
-		if (revenue < 0) { Logger::error("Negative revenue for province ", location.get_identifier()); }
+	if (revenue <= fixed_point_t::_0() || total_worker_count_in_province <= 0) {
+		if (revenue < fixed_point_t::_0()) { Logger::error("Negative revenue for province ", location.get_identifier()); }
 		if (total_worker_count_in_province < 0) { Logger::error("Negative total worker count for province ", location.get_identifier()); }
```

Comment:
```suggestion
		if (revenue < fixed_point_t::_0()) {
			Logger::error("Negative revenue for province ", location.get_identifier());
		}
		if (total_worker_count_in_province < 0) {
			Logger::error("Negative total worker count for province ", location.get_identifier());
		}
```
Better to always break for `{}` blocks, makes the code easier to read